units/host_status.go

```diff
@@ -107,6 +107,10 @@ clientsLoop:
 					j.AddError(j.terminateUnknownHosts(ctx, err.Error()))
 					continue clientsLoop
 				}
+				grip.Error(message.WrapError(err, message.Fields{
```

Comment:
I think we should grip.Debug this -- since we already add the error to the job, this would be double counted in the error alert.

Reply:
makes a lot of sense